includes/class-wc-gateway-stripe.php

```diff
@@ -647,4 +650,42 @@ public function process_payment( $order_id, $retry = true, $force_save_source =
 			);
 		}
 	}
+
+	/**
+	 * Displays the Stripe fee
+	 * @param $order_id
+	 */
+	public function display_order_fee( $order_id ) {
+		$fee = get_post_meta( $order_id, self::META_NAME_FEE, true );
+		if ( !$fee ) { return; }
```

Comment:
Coding standards: Missing space after ! and also inline conditions are not allowed.